logger.go

```diff
@@ -144,7 +144,7 @@ func (log *logger) Check(lvl Level, msg string) *CheckedEntry {
 	ce.ErrorOutput = log.errorOutput
 
 	if log.addCaller {
-		ce.Entry.Caller = MakeEntryCaller(runtime.Caller(log.callerSkip))
+		ce.Entry.Caller = MakeEntryCaller(runtime.Caller(log.callerSkip + 2))
```

Comment:
Well, I'd argue for leaving it in place:
- it truly is path dependent on how `logger` calls `check`
- originally I started out with it as a third arg to the new `check` method...
- ...which was either 2 or 3 depending on the entry path call site to `check`
- however, after I modified all the named entries (Debug, Info, Warn, etc) to call `check` instead of `Log`, it turned out that all the sites were passing the same `2` skip arg

I'd be happy to re-introduce a constant, but:
- it doesn't deserve to be package level any more
- rather it would be at the top of `check`, and called something like `skipOffset`

I'll do that since it'll dovetail with any future needs if we ever need to re-promote it to a third `check` arg.